lib/logstasher/rails_ext/rack/debug_exceptions.rb

```diff
@@ -0,0 +1,26 @@
+module LogStasher
+  module ActionDispatch
+    class DebugExceptions < ::ActionDispatch::DebugExceptions
+      private
+      def log_error(env, wrapper)
+        logger = logger(env)
+        return unless logger
+
+        ::ActiveSupport::Deprecation.silence do
+          logger << LogStasher.build_logstash_event(build_exception_hash(wrapper), ["exception"]).to_json + "\n"
+        end
+      end
+
+      def build_exception_hash(wrapper)
+        exception = wrapper.exception
+        trace = wrapper.application_trace
+        trace = wrapper.framework_trace if trace.empty?
+
+        { error: 
+          ({ exception: exception.class, message: exception.message, trace: trace}.
+           merge!( exception.respond_to?(:annotated_source_code) && { annotated_source_code: exception.annoted_source_code } || {} ))
```

Comment:
Agree. For the quick first try I didn't have a better idea. But the *merge* will be changed.